src/raftstore/store/worker/region.rs

```diff
@@ -303,4 +303,10 @@ impl Runnable<Task> for Runner {
             } => self.ctx.handle_destroy(region_id, start_key, end_key),
         }
     }
+
+    fn shutdown(&mut self) {
```

Comment:
seem we don't use this in this PR.

Reply:
I will be called outside automatically.

Reply:
This thread may need a long time to finish?